python-frontend/src/main/java/org/sonar/python/semantic/v2/types/TrivialTypeInferenceVisitor.java

```diff
@@ -307,17 +309,27 @@ public void visitImportName(ImportName importName) {
           .toList();
         var resolvedType = projectLevelTypeTable.getType(fqn);
 
-        if (!(resolvedType instanceof ModuleType module)) {
-          return;
-        }
-        if (aliasedName.alias() != null) {
-          setTypeToName(aliasedName.alias(), module);
-        } else {
-          for (int i = names.size() - 1; i >= 0; i--) {
-            setTypeToName(names.get(i), module);
-            module = Optional.ofNullable(module)
-              .map(ModuleType::parent)
-              .orElse(null);
+
+        if (resolvedType instanceof ModuleType module) {
+          if (aliasedName.alias() != null) {
+            setTypeToName(aliasedName.alias(), module);
+          } else {
+            for (int i = names.size() - 1; i >= 0; i--) {
+              setTypeToName(names.get(i), module);
+              module = Optional.ofNullable(module)
+                .map(ModuleType::parent)
+                .orElse(null);
+            }
+          }
+        } else if (resolvedType instanceof UnknownType || (resolvedType instanceof ObjectType objectType && objectType.type() instanceof UnknownType)) {
```

Comment:
Do we get `ObjectType[UnknownType]` somewhere? Is it expected? We probably want to avoid it (should we create a ticket?)

Reply:
We do get them.
They are probably coming from the converters and the systematic wrapping in ObjectType. 
I think right now, it is unexpected and is happening for the wrong reasons.
I will create a ticket

In the future, I think we might have some. As long as we know for sure it is an object.